src/themes/theme_jlmke.jl

```diff
@@ -0,0 +1,69 @@
+function theme_jlmke()
+    function jlmke_colors(alpha=1.0)
+        colors = [RGB(0.0, 0.0, 0.0), RGB(0.082, 0.643, 0.918), RGB(0.91, 0.122, 0.361),
+                  RGB(0.929, 0.773, 0.0), RGB(0.588, 0.196, 0.722), RGB(0.361, 0.722, 0.361),
+                  RGB(0.522, 0.522, 0.522)]
+        @. RGBAf(red(colors), green(colors), blue(colors), alpha)
+    end
```

Comment:
```suggestion
function jlmke_colors(alpha=1.0)
    colors = [RGB(0.0, 0.0, 0.0), RGB(0.082, 0.643, 0.918), RGB(0.91, 0.122, 0.361),
              RGB(0.929, 0.773, 0.0), RGB(0.588, 0.196, 0.722), RGB(0.361, 0.722, 0.361),
              RGB(0.522, 0.522, 0.522)]
    @. RGBAf(red(colors), green(colors), blue(colors), alpha)
end

"""
    theme_jlmke()::Attributes

A theme by Lazaro Alonso (`@lazarusa`).
Code to set the theme to `theme_jlmke` is as follows: `Makie.set_theme!(theme_jlmke())`.

The first 3 colours in the palette definition are close to the ones used in the logo of Makie. The other two are close to those of the Julia logo.
"""
function theme_jlmke()
```

Added a small docstring and some tips, hope they make sense to have here.

Reply:
thanks.